### ISSUE TITLE
Enable extension collections in Beta

### ISSUE THREAD

**Issue by cpeterso** [mozilla-mobile/fenix#26948]
Follow up from https://github.com/mozilla-mobile/fenix/issues/14034

Similar to how Nightly functions, we want to enable custom collections to be used in Firefox Beta as well.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21548)

**Comment by parcelcat** [comment]
This feature request has received extremely positive feedback on Mozilla Connect, with 237 votes and counting:

https://connect.mozilla.org/t5/ideas/expand-add-on-compatibility-in-firefox-for-android/idi-p/191

**Comment by claudiu64** [comment]
> Follow up from #14034
> 
> Similar to how Nightly functions, we want to enable custom collections to be used in Firefox Beta as well.
> 
> ┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21548)

Hello ! When will the custom collection for extensions be implemented in Firefox beta? What stage is it in? Maybe in the future work will also be done for the stable version of Firefox.

**Comment by cpeterso** [comment]
> Hello ! When will the custom collection for extensions be implemented in Firefox beta? What stage is it in? Maybe in the future work will also be done for the stable version of Firefox.

We don't have a schedule estimate, but this beta bug is now assigned to a Firefox engineer to work on. We need more testing of extensions before we can custom collections in stable Firefox.

**Comment by cpeterso** [comment]
> I noticed that it was worked on. Will the extension collection be enabled in the next Firefox Beta 106.0 b5 update?

Not yet. Support for extension collections in Beta should become available in Firefox Beta 107.0 b1 (2022-10-17). If it doesn't work then, please let us know!

**Comment by claudiu64** [comment]
> > I noticed that it was worked on. Will the extension collection be enabled in the next Firefox Beta 106.0 b5 update?
> 
> Not yet. Support for extension collections in Beta should become available in Firefox Beta 107.0 b1 (2022-10-17). If it doesn't work then, please let us know!

Thanks ! This is really welcome in Firefox Beta. 🙂 I hope you develop Firefox more. 🙂

**Comment by claudiu64** [comment]
We expect the custom collection in the stable version of Firefox as well.

**Comment by Rob--W** [comment]
> We expect the custom collection in the stable version of Firefox as well. 

This issue is specifically about enabling on Beta. That task has been completed, so the issue can be closed.

The feature request for stable exists at #19594.

**Comment by SoftVision-LorandJanos** [comment]
Verified as fixed in the latest Beta (107.0b1).
Device used: Google Pixel 6 (Android 13).
Closing the ticket as verified.

**Comment by vehosah** [comment]
Alright now add it to stable

**Comment by mdxe** [comment]
Stop hidding custom collections addon-on in the developer option of a development build at the very least...
